data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/SqsWorker.java

```diff
@@ -262,7 +264,19 @@ && isEventBridgeEventTypeCreated(parsedMessage)) {
                         },
                         Duration.ofSeconds(progressCheckInterval));
                 }
+                messageAcknowledgementSetMap.put(parsedMessage, acknowledgementSet);
+                messageWaitingForAcknowledgementsMap.put(parsedMessage, waitingForAcknowledgements);
             }
+        }
+        
+        if (endToEndAcknowledgementsEnabled) {
+            LOG.info("Created acknowledgement sets for {} messages.", parsedMessagesToRead.size());
```

Comment:
Let's make this a `debug` level log. Otherwise, we'll have too many logs generated.

Reply:
OK. I will update the log level to `debug`.